presto-cli/src/main/java/io/prestosql/cli/Console.java

```diff
@@ -127,6 +129,22 @@ public boolean run()
             }
         }
 
+        // Read queries from stdin
+        if (!hasQuery && !detectRealTerminal()) {
+            try {
+                if (System.in.available() > 0) {
+                    query = new String(ByteStreams.toByteArray(System.in), UTF_8) + ";";
```

Comment:
should we assume that locale in terminal is `UTF_8`?

Reply:
Fixed